contracts/libraries/VaultLifecycle.sol

```diff
@@ -15,6 +15,7 @@ import {
 } from "../interfaces/GammaInterface.sol";
 import {IERC20Detailed} from "../interfaces/IERC20Detailed.sol";
 import {SupportsNonCompliantERC20} from "./SupportsNonCompliantERC20.sol";
+import "hardhat/console.sol";
```

Comment:
You can use `yarn clean-logs` to get rid of these imports

Reply:
The linter should be banning console.sol